lib/all_futures/base.rb

```diff
@@ -3,9 +3,13 @@
 module AllFutures
   class Base < ActiveEntity::Base
     prepend ::AllFutures::Callbacks
-    include ::ActiveModel::Conversion
     include ::AllFutures::Persist
     include ::AllFutures::Dirty
+    include ::AllFutures::Timestamp
+    include ::ActiveModel::Conversion
+    extend ::ActiveModel::Naming
+    include ::ActiveRecord::Integration
```

Comment:
Feeling slow: this just adds the `cache_key` method? Does it do or add other things?

Are you 100% sure we can't just reimplement this method so we don't have to include `ActiveRecord` modules, which could have side effects in a library like this? I wouldn't be so uptight if there wasn't so much going on behind the scenes to make AE work.

Reply:
It's actually a pretty slim module. Apart from `cache_key` it adds `to_param` and `cache_key_with_version`:

https://github.com/rails/rails/blob/main/activerecord/lib/active_record/integration.rb